pkg/deployment/deployment.go

```diff
@@ -118,7 +124,9 @@ func Pull(apiClient *clients.Settings, name, nsname string) (*Builder, error) {
 
 // WithNodeSelector applies a nodeSelector to the deployment definition.
 func (builder *Builder) WithNodeSelector(selector map[string]string) *Builder {
-	if valid, _ := builder.validate(); !valid {
+	if valid, err := builder.validate(); !valid {
+		builder.errorMsg = err.Error()
```

Comment:
I think it is redundant. builder.errorMsg should already contain what you are registering in err

Reply:
I'll double-check what I was trying to do with this one.

Reply:
I double checked and removed it.  Good catch I'm not sure why I added that but it wasn't needed.  😄